hathor/transaction/storage/transaction_storage.py

```diff
@@ -1146,6 +1148,11 @@ def partial_vertex_exists(self, vertex_id: VertexId) -> bool:
         with self.allow_partially_validated_context():
             return self.transaction_exists(vertex_id)
 
+    def get_mempool_tips(self) -> set[VertexId]:
```

Comment:
Missing docstring.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1165/commits/b9b84ea0802ea3ae98dbc324d8ab07434a2f3575

Reply:
Thanks!